### ISSUE TITLE
qt 5.3.2 static cannot find qmake spec

### ISSUE THREAD

**Issue by adipose** [msys2/MINGW-packages#225]
After upgrading from Qt 5.3.1-static to 5.3.2-static, I get the following error when trying to run qmake:

``` bash
/C/msys2/mingw32/qt5-static/bin/qmake.exe VPATH=. -spec win32-g++ -o qttmp-Release531.mk nbproject/qt-Release531.pro
Could not find qmake configuration file win32-g++
```

The exact same command on 5.3.1 works with no errors.  After downgrading, my problem disappeared.

**Comment by adipose** [comment]
So, this came back when I updated my toolchain (it's been a few years--once working, don't update :)).  When I tried to run the post-install script, it seemed there wasn't one anymore.  So, I had to remove qt and reinstall it, which fixed it.  This seems quite common that running pacman -Syu results in a lot of broken packages that have to be reinstalled by hand.  Maybe a new option is needed to update some critical libraries first, then restart the shell, and then run update again.  I also had to reinstall gcc and toolchains 3-4 times before msys make would work properly.
